num_enum_derive/src/lib.rs

```diff
@@ -30,10 +31,60 @@ fn literal(i: u64) -> Expr {
     }
 }
 
+mod kw {
+    syn::custom_keyword!(default);
+    syn::custom_keyword!(aliases);
+}
+
+enum NumEnumVariantAttributes {
+    Default,
+    Aliases(VariantAliasesAttribute),
+}
+
+impl Parse for NumEnumVariantAttributes {
+    fn parse(input: ParseStream<'_>) -> Result<Self> {
```

Comment:
Can we tighten up this parsing a bit? Right now, it looks at the first token, but ignores, rather than errors, on things we don't expect. Some examples of things I tried out which were silently ignored:
```rust
#[num_enum(blah)]
#[num_enum(default, aliases = [2, 3])]
#[num_enum(aliases([2, 3))]
```

Reply:
Fixed. See added unit tests